src/main/java/org/elasticsearch/script/ScriptParameterParser.java

```diff
@@ -231,11 +231,4 @@ public ScriptType scriptType() {
             return scriptType;
         }
     }
-
-    public static class ScriptParameterParseException extends ElasticsearchException {
```

Comment:
yeah, I think in 1.6 we should just keep on using the old exception and switch to the new one in 2.0